### PR TITLE
EWPP-476: scaffold module.

### PR DESCRIPTION
**Pull request by brummbar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,101 @@
-#multivalue_form_element
+# OpenEuropa multivalue_form_element
```
Comment (hernani) [inline README.md:1]: Should we remove OpenEuropa here?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,101 @@
...
 
-Form element that wraps other simple form elements to make them multi-value.
+The OpenEuropa multivalue_form_element provides ...`[One Paragraph of project description goes here]`
```
Comment (hernani) [inline README.md:3]: Same.

File: multivalue_form_element.module [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa module.
```
Comment (hernani) [inline multivalue_form_element.module:5]: Fix?

File: README.md (hunk 1 above)
Comment (hernani) [reply README.md:1]: Fixed in following.

File: README.md (hunk 2 above)
Comment (hernani) [reply README.md:3]: Fixed in following.

File: multivalue_form_element.module (hunk 3 above)
Comment (hernani) [reply multivalue_form_element.module:5]: Removed later.

**Review by hernani — APPROVED** [review]

**Review by imanoleguskiza — APPROVED** [review]

**Pull request merged**